dial_delay.go

```diff
@@ -0,0 +1,208 @@
+package swarm
+
+import (
+	"context"
+	"time"
+
+	ma "github.com/multiformats/go-multiaddr"
+	mafmt "github.com/whyrusleeping/mafmt"
+)
+
+const p_circuit = 290
+
+const numTiers = 2
+
+var tierDelay = 2 * time.Second
```

Comment:
make this exported, we will want to be able to tweak it easily i think

Reply:
done